docker-compose.yml

```diff
@@ -9,6 +9,7 @@ x-app: &common
     BUNDLE_ENTERPRISE__CONTRIBSYS__COM: "${BUNDLE_ENTERPRISE__CONTRIBSYS__COM}"
     SETTINGS__DATABASE_URL: "postgis://${POSTGRES_USER:-postgres}:${POSTGRES_PASSWORD:-password}@${POSTGRES_HOST:-postgres}:${POSTGRES_PORT:-5432}/${POSTGRES_DATABASE:-vets_api_development}?pool=4"
     SETTINGS__TEST_DATABASE_URL: "postgis://${POSTGRES_USER:-postgres}:${POSTGRES_PASSWORD:-password}@${POSTGRES_HOST:-postgres}:${POSTGRES_PORT:-5432}/${POSTGRES_DATABASE:-vets_api_test}"
+    SETTINGS__AUDIT_DB__URL: "postgis://${POSTGRES_USER:-postgres}:${POSTGRES_PASSWORD:-password}@${POSTGRES_HOST:-postgres}:${POSTGRES_PORT:-5432}/${POSTGRES_DATABASE:-vets_api_audit}"
```

Comment:
Do we need the test version defined in any of these docker compose files?

Reply:
No, I made it so development and test use the same setting `audit_db.url` and just append `development` or `test`.

[`settings.yml`](https://github.com/department-of-veterans-affairs/vets-api/blob/bda4473cdf8c5c3b842b4f603fb605b5eb29e3e4/config/settings.yml#L12)
```yml
audit_db:
  url: postgis:///vets_api_audit
```

[`database.yml`](https://github.com/department-of-veterans-affairs/vets-api/blob/bda4473cdf8c5c3b842b4f603fb605b5eb29e3e4/config/database.yml#L10)
```yml
development:
  audit:
    url: <%= "#{Settings.audit_db.url}_development" %>

test:
  audit:
    url: <%= "#{Settings.audit_db.url}_test#{ENV['TEST_ENV_NUMBER']}" %>
```